src/scene/graphics/shared/FillTypes.ts

```diff
@@ -39,6 +39,8 @@ export interface StrokeAttributes
     join?: LineJoin;
     /** The miter limit to use. */
     miterLimit?: number;
+    /** If the stroke is a pixel line. */
+    pixelLine?: boolean;
```

Comment:
This type is also used by `Text`/`BitmapText` for its stroke properties, so we should omit this in `TextStyle.stroke`

Reply:
good shout!